common/src/app/common/files/changes_builder.cljc

```diff
@@ -1135,3 +1135,17 @@
 (defn get-page-id
   [changes]
   (::page-id (meta changes)))
+
+
+(defn set-base-font-size
+  [changes new-base-font-size]
+
+  (let [file-data  (::file-data (meta changes))
```

Comment:
Metadata in changes is optional. If some builder needs `::file-data`, the caller must have added `with-file-data` before calling this builder, and this one should add an `(assert-file-data! changes)` to ensure this.

In this case the `assert-file-data` function does not exist. You should create it, and check that `::file-data` key exists in meta.

Bonus action is to add asserts to other changes that currently are missing, like `set-plugin-data` (`assert-file-data`), `set-guide`, `set-flow`, `set-comment-thread-position` or `set-default-grid` (`assert-page`).